conda/models/index_record.py

```diff
@@ -81,30 +56,100 @@ class Link(DictSafeMixin, Entity):
 EMPTY_LINK = Link(source='')
 
 
-class FeaturesField(ListField):
+def push_individual_feature(result_map, val):
+    k, v = translate_feature_str(val)
+    result_map[k] = v
+
+
+def _make_provides_features(track_features, instance):
```

Comment:
Ah, but _that_ is different than just pinning the version of Python; that's a different Python _variant_. Even then, let's try to leave version numbers out of the variants when we can (might not be possible with pypy, I understand).